m34thook/memscan.hpp

```diff
@@ -653,7 +655,11 @@ struct scangroup_t {
 #ifdef ASSERT_ALL_LOCATED
 	template<typename T, typename... TRest>
 	static void assert_all_located() {
-		assert(*(T::g_blockscan_node.m_result_receiver));
+		if(!*(T::g_blockscan_node.m_result_receiver)) {
```

Comment:
@id-sechs69 These are the scanner failed message. Which were originally an assert.